tests/integration/test_smoothing.py

```diff
@@ -37,16 +37,16 @@ class DescribeSliceSmoothing(object):
     def it_provides_smoothed_col_pct_for_compatible_cubes(
```

Comment:
Remove lines 4-5 above, now that we're all on Python3 for development we don't need that in tests anymore.

Reply:
Remove lines 12-13 about mnemonics. TR is not used in this module and we all recognize CR for what it is now so these are just noise.